components/reusable-components/add-skill/AddSkills.tsx

```diff
@@ -0,0 +1,57 @@
+'use client';
+
+import { useState } from 'react';
+import styles from './AddSkill.module.scss';
+import TagsModal from '../tags-modal/TagsModal';
+
+const existingTags = [
+  'JavaScript',
+  'React',
+  'Next.js',
+  'CSS',
+  'HTML',
+  'Node.js',
+  'Java',
+  'C#',
+  'PHP',
+];
+
+const AddSkill = () => {
+  const [showModal, setShowModal] = useState(false);
+  const [selectedTags, setSelectedTags] = useState<string[]>([]);
+  const openModal = () => setShowModal(true);
+  const closeModal = () => setShowModal(false);
+
+  return (
+    <div className={styles.addSkillWrapper}>
+      <button type="button" onClick={openModal} className={styles.addButton}>
+        + Add Skill
+      </button>
+      {showModal && (
+        <TagsModal
+          existingTags={existingTags}
+          selectedTags={selectedTags}
+          setSelectedTags={setSelectedTags}
+          onClose={closeModal}
+        />
+      )}
+      <div className={styles.selectedTags}>
+        {/* eslint-disable jsx-a11y/no-static-element-interactions */}
+        {/* eslint-disable jsx-a11y/click-events-have-key-events */}
```

Comment:
This eslint errors are easy to fix lets do that instead of disabling the rules:
1. Convert the span element to button
2. Add role to the button
3. Add aria-label = 'Remove' or aria-label='Delete'
more information on why we need to add aria-labels to help with assistive technology: https://stackoverflow.com/questions/22039910/what-is-aria-label-and-how-should-i-use-it

The problem with using span here is that it is non-native interactive element. We need to make sure we are using semantic html.